lib/WorseReflection/Tests/Integration/Bridge/TolerantParser/Diagonstics/UnusedImportProvider/UsedByDoctrineAttributeAliased.test

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use Doctrine\Mapping as ORM;
+
+#[ORM\Id]
```

Comment:
attributes _should_ be detected in the AST not with regex

Reply:
oh, maybe this is the real problem:
![image](https://user-images.githubusercontent.com/42226341/181916588-ed60c5f4-126f-49aa-866e-8e23878e5617.png)

Reply:
ready to merge